src/rpm/builder.rs

```diff
@@ -168,7 +168,7 @@ impl RPMBuilder {
     ///
     /// ```
     /// # fn foo() -> Result<(), Box<dyn std::error::Error>> {
-    /// let pkg = rpm::RPMBuilder::new("foo", "1.0.0", "MPL", "x86_64", "some bar package")
+    /// let pkg = rpm::RPMBuilder::new("foo", "1.0.0", "MIT", "x86_64", "some bar package")
```

Comment:
That wasn't a typo :)

Reply:
It's not a correct SPDX identifier though, I could change it to `MPL-2.0`

Reply:
Yes please